team/bundles/org.eclipse.jsch.ui/src/org/eclipse/jsch/internal/ui/preference/PreferencePage.java

```diff
@@ -485,6 +500,9 @@ public void widgetSelected(SelectionEvent e){
 					if(e.widget==keyGenerateDSA){
 						type=KeyPair.DSA;
 						_type=IConstants.DSA;
+						if (keySizeValue.getSelection() > DSA_KEY_SIZE) {
+							keySizeValue.setSelection(DSA_KEY_SIZE);
```

Comment:
Adapt to new constant names
```suggestion
						if (keySizeValue.getSelection() > DSA_MAX_KEY_SIZE) {
							keySizeValue.setSelection(DSA_MAX_KEY_SIZE);
```